Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
+
+- Bump MongoDB.Driver from 2.13.1 to 2.15.0 and Microsoft.AspNetCore.Http from 2.1.0 to 2.2.0. [#608](https://github.com/microsoft/sarif-pattern-matcher/pull/608)
```

Comment:
Should the CVEs be linked here?  I notice most all release line items are very factual and explain just the change and not rationale as to the why.  Is that is intended I suppose CVEs don't need to be historically captured here.

Reply:
One of my first iterations I had added, then I removed to simplify and added this to the PR.

The rationale: you as a customer who is using will only see the release notes if you face a problem. For things like vulnerabilities, I think you wouldn't come, so, it's why I omitted in the end.